apps/mosaico/src/Main.purs

```diff
@@ -224,6 +232,33 @@ frontpage { htmlTemplate } _ = do
             appendMosaico mosaicoString htmlTemplate >>= appendHead windowVars
   pure $ TextHtml html
 
+tagList :: Env -> { params :: { tag :: String }, guards :: { credentials :: Maybe UserAuth } } -> Aff TextHtml
+tagList { htmlTemplate } { params: { tag } } = do
+  articles <- Lettera.getByTag 0 10 (uriComponentToTag tag) HBL
```

Comment:
Maybe we could have a bit more by default, maybe 20?

Reply:
I bumped it to 20.